src/app/static/src/app/components/password/ForgotPassword.vue

```diff
@@ -1,26 +1,34 @@
 <template>
     <div class="card reset-password-form mx-auto mt-5">
         <div class="card-body">
-            <h3 class="card-title">Forgotten your password?</h3>
-            <p>
-                If you've forgotten your password, enter your email address to request a link which you can use
-                to create a new password.
+            <h3 class="card-title" v-translate>
+                forgottenPassword
```

Comment:
So I've actually had to introduce exactly this syntax:
`v-translate=keyName` elsewhere now, because `<div v-translate>keyName</div>` only works where `keyName` is static, i.e. `<div v-translate>{{keyName}}</div>` won't work. If you think the former is unintuitive anyway lets just go with the latter everywhere then :+1: